### PR TITLE
Trainee site done

### PR DESCRIPTION
**Pull request by jevikraiyani12**

### CODE REVIEW

**Comment by jevikraiyani12** [comment]
REFS: #265 and fixes #120

**Comment by jevikraiyani12** [comment]
REFS #120 and fixes #120

**Review by radixdt12020 — CHANGES_REQUESTED** [review]
Add more fields in car sales business

File: Modules/SQL/Day1/Practical-Assignment/SQLQuery1.sql [hunk 1]
```diff
@@ -0,0 +1,144 @@
...
+
+
+		CREATE TABLE dbo.Jobs
+		(
+			JobId int CONSTRAINT pkJobId PRIMARY KEY,
+			PersonId int CONSTRAINT fkPersonId FOREIGN KEY REFERENCES dbo.Persoons(PersonId),
+		   [Location] varchar(50) chkLocation CHECK(Location IN('Ahmedabad','Rajkot','Gandhinagar'))
+		)
+
+		AddressPerson varchar(30) chkAddressPerson C
```
Comment (radixdt12020) [inline Modules/SQL/Day1/Practical-Assignment/SQLQuery1.sql]: incomplete

**Review by radixdt12020 — CHANGES_REQUESTED** [review]
Data type should be capital

**Comment by jevikraiyani12** [comment]
REFS:120 and fixes #120

File: Modules/SQL/Day1/jevik-raiyani/Practical-Assignment/SQLQuery1.sql [hunk 2]
```diff
@@ -1,144 +1,136 @@
...
 
 CREATE TABLE dbo.jobs 
 (
-	JobId int CONSTRAINT pkjobId PRIMARY KEY,
-	JobTitle varchar(30) CONSTRAINT chkDefJobTitle DEFAULT '  ',
-	MinSalary int CHECK (MinSalary> 8000) NOT NULL,
-	MaxSalary int
+	JobId INT CONSTRAINT pkjobId PRIMARY KEY,
+	JobTitle VARCHAR(30) CONSTRAINT chkDefJobTitle DEFAULT '  ',
+	MinSalary INT CHECK (MinSalary> 8000) NOT NULL,
```
Comment (radixdt12020) [inline Modules/SQL/Day1/jevik-raiyani/Practical-Assignment/SQLQuery1.sql:91]: default value should be 8000 
this constraint is missing

**Pull request merged**